BREAKING_CHANGES.md

```diff
@@ -1,5 +1,43 @@
 # Breaking Changes
 
+## 0.6.0
+
+- The pytest rules `PT001` and `PT023` now default to omitting the decorator parentheses when there are no arguments ([#12838](https://github.com/astral-sh/ruff/pull/12838)).
```

Comment:
Here, too, I'd personally put this bullet point last out of the three, since these rules have safe autofixes. But just a nit.

Reply:
I prefer to keep the long notebook section last. But I can move it further down if you insist ;)